src/tests/Microsoft.PowerFx.Interpreter.Tests/CollectFunctionTests.cs

```diff
@@ -61,7 +62,7 @@ public async Task AppendCountTest(string script, int expected)
         [InlineData("Collect(lazyTable, lazyCoercibleRecord)", true)]
         [InlineData("Collect(lazyTable, lazyNotCoercibleRecord)", false)]
         [InlineData("Collect(lazyTable, {Value:1})", false)]
-        [InlineData("Collect(lazyTable, lazyTable)", false)]
+        [InlineData("Collect(lazyTable, lazyTable)", true)]
```

Comment:
>true [](http://example.com/codeflow?start=53&length=4)

nice ...  what flipped this?

Reply:
We now support all possible overloads:
`Collect(collection:*[...], item1:![...]|*[...], ...)`
`Collect(collection:*[...], item1, ...)`